### DISCUSSION TITLE
Targets cookbook?

### DISCUSSION THREAD

**Discussion by liutiming** [ropensci/targets#572] [Ideas]
Hi Will, have you thought about whether to start a project on a targets cookbook?

**Comment by wlandau** [comment]
I did consider it, but I prefer [individual repos](https://docs.ropensci.org/targets/#example-projects) for examples because they are easier for people to try out.

**Reply by liutiming** [reply]
Yup, perhaps what I was thinking was more of a summary of the use cases that people have reported on issues/discussions - if we can group them into themes like the rmarkdown/ggplot cook book, then it might be more accessible
